clients/csharp-wpf/VoiceAssistantClient/Settings/ConnectionProfile.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
```

Comment:
Do the using directives need to go inside of the namespace? I get StyleCop warnings in projects when I put them at the top of the file.

Reply:
Yes, using directives must go inside of the namespace. That using statement is not used - will remove it